web/app/integration/wdio-sauce.conf.js

```diff
@@ -17,12 +26,12 @@ exports.config = {
   ],
   bail: 0,
   baseUrl: 'http://localhost',
-  waitforTimeout: 10000,
-  connectionRetryTimeout: 90000,
+  waitforTimeout: 1000000,
+  connectionRetryTimeout: 90000000,
```

Comment:
These timeouts seem a little high. `waitforTimeout` looks like it waits for 16 minutes, while `connectionRetryTimeout` waits for 25 hours. I'm wondering if you were running into timing issues with the test? 

I think we could set these to something much lower, or maybe use the default values supplied by webdriverIO. We would like to make sure we aren't spending too much time running tests since the goal is to have tests run as quickly as possible to get feedback quickly when we encounter a bug in the UI.

Reply:
Ah, my apologies, forgot to remove some of the zeros. The default values tend to be too low for successful running of tests, especially when using browser.waitUntil(), browser.pause() on sauce labs (and on slower connections) so I added a few(too many) extra zeros to fix it. I'll change those values to make them more suitable. So sorry about that. Thank you for pointing it out!